README.md

```diff
@@ -1,9 +1,14 @@
+[![Build Status](https://travis-ci.org/angular/angular.svg?branch=master)](https://travis-ci.org/inveniosoftware-contrib/ng2-json-editor)
```

Comment:
I think it should be `https://travis-ci.org/inveniosoftware-contrib/ng2-json-editor.svg?branch=master` instead of `https://travis-ci.org/angular/angular.svg?branch=master`